.github/workflows/codeql.yml

```diff
@@ -0,0 +1,44 @@
+name: "Code Scanning - Action"
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
any specific reason to use cron scheduling?

Reply:
We have the same implementation in another our PR
https://github.com/microsoft/appcenter-sdk-react-native/pull/826
I think we need to keep uniformity anywhere.